VATSpy.dat

```diff
@@ -18273,36 +18273,38 @@ LRBB|Bucharest||LRBB
 LSAS|Swiss Radar - Switzerland||LSAS
 LSAG|Swiss Radar (Geneva ACC) - Switzerland||LSAG
 LSAZ|Swiss Radar (Zurich ACC) - Switzerland||LSAZ
-LTAA|Ankara Control - Ankara|ANK|LTAA
-LTAA-A|Ankara Control - Ankara Central & East|ANK_A|LTAA-A
-LTAA-A|Ankara Control - Ankara Central|ANK_C|LTAA-A
-LTAA-A|Ankara Control - Ankara East|ANK_E|LTAA-A
-LTAA-A|Ankara Control - Ankara North|ANK_N|LTAA-A
-LTAA-A|Ankara Control - Ankara South|ANK_S|LTAA-A
-LTAA-A|Ankara Control - Ankara East North|ANK_EN|LTAA-A
-LTAA-A|Ankara Control - Ankara East South|ANK_ES|LTAA-A
-LTAA-A|Ankara Control - Ankara 7 Sector|ANK_7|LTAA-A
-LTAA-A|Ankara Control - Ankara 8 Sector|ANK_8|LTAA-A
-LTAA-A|Ankara Control - Ankara 9 Sector|ANK_9|LTAA-A
-LTAA-A|Ankara Control - Ankara 10 Sector|ANK_10|LTAA-A
-LTAA-A|Ankara Control - Ankara 11 Sector|ANK_11|LTAA-A
-LTAA-A|Ankara Control - Ankara 12 Sector|ANK_12|LTAA-A
-LTAA-A|Ankara Control - Ankara 13 Sector|ANK_13|LTAA-A
-LTAA-A|Ankara Control - Ankara 14 Sector|ANK_14|LTAA-A
-LTAA-A|Ankara Control - Ankara 15 Sector|ANK_15|LTAA-A
-LTAA-A|Ankara Control - Ankara 16 Sector|ANK_16|LTAA-A
-LTAA-A|Ankara Control - Ankara 17 Sector|ANK_17|LTAA-A
-LTAA-A|Ankara Control - Ankara 18 Sector|ANK_18|LTAA-A
-LTAA-A|Ankara Control - Ankara 19 Sector|ANK_19|LTAA-A
-LTBB|Ankara Control - Istanbul|ANK_W|LTBB
-LTBB|Ankara Control - Istanbul West North|ANK_WN|LTBB
-LTBB|Ankara Control - Istanbul West South|ANK_WS|LTBB
-LTBB|Ankara Control - Ankara 1 Sector|ANK_1|LTBB
-LTBB|Ankara Control - Ankara 2 Sector|ANK_2|LTBB
-LTBB|Ankara Control - Ankara 3 Sector|ANK_3|LTBB
-LTBB|Ankara Control - Ankara 4 Sector|ANK_4|LTBB
-LTBB|Ankara Control - Ankara 5 Sector|ANK_5|LTBB
-LTBB|Ankara Control - Ankara 6 Sector|ANK_6|LTBB
+LTAA|Ankara||LTAA
+LTAA-A|Ankara Control - Ankara Central & East|ANK_A|LTAA

```

Comment:
Missing `-A` at the very end

Reply:
Hello Adrian. this needs to stay like this; because with my PR, LTAA-A polygon will be renamed as LTAA.
I already tested this and working very good.

while offline:
![image](https://github.com/user-attachments/assets/0a04c3d4-b942-4882-9195-5582291820ff)

while ANK_A_CTR online:
![image](https://github.com/user-attachments/assets/cbabae23-f6ba-475d-a4d7-f8aed1e2da28)